concrete/blocks/page_attribute_display/controller.php

```diff
@@ -104,6 +106,19 @@ public function getRequiredFeatures(): array
         ];
     }
 
+    public function cacheBlockOutputForRegisteredUsers()
+    {
+        if ($this->btCacheBlockOutputForRegisteredUsers === null) {
+            $templateHandle = $this->getTemplateHandle();
+            /**
+             * Date helper may change output depends on the user's timezone or language, so we can't cache output.
+             * Otherwise, we can cache output.
+             */
+            $this->btCacheBlockOutputForRegisteredUsers = $templateHandle !== 'date_time';
```

Comment:
This is kind of bad because it will break other peoples template, especially if they have attributes that are dates but not one of the following:
```
"rpv_pageDateCreated"
"rpv_pageDateLastModified"
"rpv_pageDatePublic"
```
or even any attribute that relies on the existing behaviour of not caching